NitroxClient/MonoBehaviours/Overrides/MultiplayerBuilder.cs

```diff
@@ -943,7 +943,8 @@ private static void CreatePowerPreview(TechType constructableTechType, GameObjec
             string poweredPrefabName = CraftData.GetPoweredPrefabName(constructableTechType);
             if (poweredPrefabName != string.Empty)
             {
-                gameObject = PrefabDatabase.GetPrefabForFilename(poweredPrefabName);
+                IPrefabRequest asdf = PrefabDatabase.GetPrefabForFilenameAsync(poweredPrefabName);
+                asdf.TryGetPrefab(out gameObject);
```

Comment:
Temporary variable name?
```suggestion
                IPrefabRequest poweredPrefabRequest = PrefabDatabase.GetPrefabForFilenameAsync(poweredPrefabName);
                poweredPrefabRequest.TryGetPrefab(out gameObject);
```